common/djangoapps/student/views.py

```diff
@@ -350,6 +352,22 @@ def _cert_info(user, course, cert_status):
         else:
             status_dict['download_url'] = cert_status['download_url']
 
+            # getting linkedin URL and then pass the params which appears
+            # on user profile. if linkedin config is empty don't show the button.
+
+            current_config = LinkedInUrlConfiguration.current()
+            if current_config.enabled:
+                cert_name = u'{type} Certificate for {cert_name}'.format(
+                    type=cert_status["mode"].title(), cert_name=course.display_name
+                ).encode('utf-8')
+
+                params_dict = {
+                    'pfCertificationName': cert_name,
+                    'pfCertificationUrl': cert_status['download_url']
+                }
+
+                status_dict['linked_in_url'] = current_config.linkedin_url + "&" + urlencode(params_dict)
```

Comment:
Please use `.format` here instead of "+" and make sure that the string is unicode (prefixed with a `u`)